models/linkages/R/read.restart.LINKAGES.R

```diff
@@ -0,0 +1,26 @@
+read.restart.LINKAGES <- function(outdir,run.id,time,IC,prior,spinup){
```

Comment:
@araiho it's been a long time since we discussed the design of read.restart.MODEL. Could you remind me why IC is passed?